### PR TITLE
Opened app

### PR DESCRIPTION
**Pull request by aberllin**
add router, so when you click on the app card, window with app open in current page without loading

### CODE REVIEW

**Comment by arevej** [comment]
https://deploy-preview-3--optimistic-fermat-7ff6ec.netlify.app

File: src/components/AppCard.js [hunk 1]
```diff
@@ -1,5 +1,74 @@
...
+    </CardContainer>
+  )
+}
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: ${({ theme }) => theme.textColor};
+`
+
+const AppWindow = ({ title, app }) => {
```
Comment (arevej) [inline src/components/AppCard.js]: I think AppWindow and CloseButton should be in a separate file

**Review by ruslanakholod — COMMENTED** [review]
image

File: src/components/AppCard.js (hunk 1 above)
Comment (aberllin) [reply src/components/AppCard.js]: fixed it

Comment (arevej) [reply src/components/AppCard.js]: Good, take a look at @ruslanakholod comment

File: src/components/AppWindow.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  return (
+    <AppWindowWrapper>
+      <AppCardNav>
+        <div>{title}</div>
+        <CloseAppButton />
+      </AppCardNav>
+      <div>{app}</div>
+    </AppWindowWrapper>
+  )
+}
```
Comment (ruslanakholod) [inline src/components/AppWindow.js]: you can write like that

```
export const AppWindow = ({ title, app }) => (
    <AppWindowWrapper>
      <AppCardNav>
        <div>{title}</div>
        <CloseAppButton />
      </AppCardNav>
      <div>{app}</div>
    </AppWindowWrapper>
  )
```

File: src/components/CloseAppButton.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import { Link } from 'react-router-dom'
+import { IoCloseCircleOutline } from 'react-icons/io5'
+
+export const CloseAppButton = () => {
+  return (
+    <Link to="/">
+      <CloseIcon />
+    </Link>
+  )
+}
```
Comment (ruslanakholod) [inline src/components/CloseAppButton.js]: same https://github.com/aberllin/react-apps-dashboard/pull/3/commits/acc85feeb602be4aad0f8aba2a567be8656222b8#r552405695

Comment (ruslanakholod) [reply src/components/CloseAppButton.js]: This component is used only in AppWindow, so you can create a folder components/AppWindow and move this component there

**Review by ruslanakholod — COMMENTED** [review]
image
i think these paddings are too big

**Review by ruslanakholod — COMMENTED** [review]
Screenshot 2021-01-06 at 10 19 04

**Comment by arevej** [comment]
Omg @ruslanakholod is on fire 🔥🔥🔥

**Pull request merged**